### PR TITLE
[CRT_APITEST] Add tests for __rt_sdiv/udiv/sdiv64/udiv64 on ARM

### PR DESCRIPTION
**Pull request by binarymaster**
Addendum to f2bc1f0e and f1e2c6cb.

### CODE REVIEW

**Review by binarymaster — COMMENTED** [review]
Although these tests are intended for ARM architecture, I also tested them compiled for x86 architecture with Visual Studio. There are no failures.

```
__rt_sdiv: 24 tests executed (0 marked as todo, 0 failures), 0 skipped.

__rt_sdiv64: 48 tests executed (0 marked as todo, 0 failures), 0 skipped.

__rt_udiv: 6 tests executed (0 marked as todo, 0 failures), 0 skipped.

__rt_udiv64: 12 tests executed (0 marked as todo, 0 failures), 0 skipped.
```

File: modules/rostests/apitests/crt/testlist.c [hunk 1]
```diff
@@ -7,6 +7,10 @@
 extern void func__vscprintf(void);
 extern void func__vscwprintf(void);
 extern void func_atexit(void);
+extern void func___rt_sdiv(void);
```
Comment (tkreuzer) [inline modules/rostests/apitests/crt/testlist.c:10]: Move up to where stuff wig `` __`` is?

File: modules/rostests/apitests/crt/msvcrt_crt_apitest.cmake [hunk 2]
```diff
@@ -1371,6 +1371,13 @@ elseif(ARCH STREQUAL "amd64")
     #    sinf.c
     #    sqrtf.c
     )
+elseif(ARCH STREQUAL "arm")
+    list(APPEND SOURCE_MSVCRT
+        __rt_sdiv.c
+        __rt_udiv.c
+        __rt_sdiv64.c
+        __rt_udiv64.c
```
Comment (tkreuzer) [inline modules/rostests/apitests/crt/msvcrt_crt_apitest.cmake:1379]: Is this really worth 4 tests? It's all just software division.

File: modules/rostests/apitests/crt/testlist.c (hunk 1 above)
Comment (binarymaster) [reply modules/rostests/apitests/crt/testlist.c]: Moved

File: modules/rostests/apitests/crt/msvcrt_crt_apitest.cmake (hunk 2 above)
Comment (binarymaster) [reply modules/rostests/apitests/crt/msvcrt_crt_apitest.cmake]: Ok I merged everything into a single test.

**Pull request merged**